.pipelines/pr-e2e.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
     k8sRelease: '1.18'
     apimodel: 'examples/e2e-tests/kubernetes/release/default/definition.json'
     createVNET: true
-    enableKMSEncryption: true
+    enableKMSEncryption: false
```

Comment:
We don't want to test this as part of PR gate anymore?

Reply:
The new implementation brings excess cloud-init data overhead, which exceeds the allowed (64k bytes) limit when incorporated into the cluster config we're testing.

So we've added a discrete KMS-enabled cluster config to the set of jenkins-enabled cluster configs we test out of band.

Reply:
Let's document how to run the kms-enabled jenkins job.